mesh/pmesh.hpp

```diff
@@ -343,7 +343,7 @@ class ParMesh : public Mesh
        meshes and quick space-filling curve equipartitioning for nonconforming
        meshes (elements of nonconforming meshes should ideally be ordered as a
        sequence of face-neighbors). */
-   ParMesh(MPI_Comm comm, Mesh &mesh, int *partitioning_ = NULL,
+   ParMesh(MPI_Comm comm, Mesh &mesh, const int *partitioning_ = NULL,
```

Comment:
For consistency with the remaining variables, can we rename partitioning_ to partitioning and use nullptr?

Reply:
But partitioning is not shadowed here, because it is not a member of the class, right?

Reply:
`partitioning` is still used in the local scope so I think keeping this as `partitioning_` is needed. See  [aab273b](https://github.com/mfem/mfem/pull/4190/commits/aab273b303873553f6d7533f4491cbcc3c54d608) which addressed [this comment](https://github.com/mfem/mfem/pull/4190#discussion_r1598081303) but also made it so the local `partitioning` variable is needed.

Reply:
Oh, I see. Then it's okay. Thanks for pointing this out.